thanos/values.yaml

```diff
@@ -64,13 +66,19 @@ compact:
   logLevel: debug
   securityContext: {}
   resources: {}
+  extraArgs:
```

Comment:
Please remove these extraArgs because of #908

Reply:
Thanks, but I had a confusion with the #908 PR. I removed these changes from it - it should be merged here, as this is how it was tested.